src/test/java/com/training/assignment/tests/AssignmentTest.java

```diff
@@ -165,4 +165,87 @@ public void testAbstractInterfaceConcept() {
         softAssert.assertThat(new Yes2().getSum(5))
                 .as("Sum of three number is incorrect").isEqualTo(35);
     }
+
+    @Test
+    public void testStaticExample() {
+        String name = "Shri";
+        String lastName = "Shetti";
+        int rollNo = 56;
+        String schoolName = "SVP";
+
+        Student student1 = new Student();
+        Student student2 = new Student();
+
+        student1.setName(name);
+        student1.setLastName(lastName);
+        student1.setRollNo(rollNo);
+        Student.setSchoolName("Anudatt");
+
+        student2.setSchoolName(schoolName);
+
+        softAssert.assertThat(student1.getName()).as("Name is not matching").isEqualTo(name);
+
+        softAssert.assertThat(student1.getLastName()).as("Lastname is not matching").isEqualTo(lastName);
+
+        softAssert.assertThat(student1.getRollNo()).as("Roll number doesn't match").isEqualTo(rollNo);
+
+        softAssert.assertThat(Student.getSchoolName()).as("School name doesn't match").isEqualTo(schoolName);
+
+        softAssert.assertThat(student2.getSchoolName()).as("School name doesn't match").isEqualTo(schoolName);
+    }
+
+    @Test
+    public void testMiniProject() {
+        EmployeeOperation empOperation = new EmployeeOperation();
```

Comment:
lets have consistency in naming. Use the object name = classname, with camel casing